UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/Construction/BuildMenuEntryController.cs

```diff
@@ -94,7 +94,17 @@ public void OnClick()
 				return;
 			}
 
-			RequestBuildMessage.Send(entry, buildingMaterial);
+			int NumberWanted = 1;
+
+
+			if (int.TryParse(UIManager.BuildMenu.NumberInputField.text, out NumberWanted) == false)
+			{
+				NumberWanted = 1;
```

Comment:
This assignment of NumberWanted is unnecessary? Already set to one above

Reply:
Just catch the out settings to a funny number?

Reply:
Oh yeah misread